src/App.jsx

```diff
@@ -1,5 +1,11 @@
+import { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { GoodList } from './components/goodList';
+
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LANGTH = 'langth';
```

Comment:
There is a typo in the constant name `SORT_FIELD_LANGTH`. It should be `SORT_FIELD_LENGTH` to correctly represent the sorting by length.